### ISSUE TITLE
Browserify clobbers node builtins

### ISSUE THREAD

**Issue by gravaton** [nexe/nexe#297]
I'm attempting to use nexe to package an application that uses node's 'cluster' module.  Browserify seems to replace this module with an empty shim because it doesn't believe that such a thing exists in the browser.  While it might not, I'm running this application in node!

The error reported is as follows:

```
cluster.setupMaster(resolverConfig);
        ^

TypeError: cluster.setupMaster is not a function
    at Object.<anonymous> (nexe.js:227:9)
    at Object.__dirname.2.../package.json (nexe.js:235:4)
    at s (nexe.js:1:316)
    at e (nexe.js:1:487)
    at nexe.js:1:505
    at NativeModule.compile (bootstrap_node.js:528:7)
    at Function.NativeModule.require (bootstrap_node.js:469:18)
    at Function.Module._load (module.js:427:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
```

It seems like Browserify is only used to package everything up into one file - maybe there's a better way to do this?

**Comment by jbackman** [comment]
https://github.com/zsoltszabo/node-uglifier looks like it could be an alternative to Browserify and might not clobber built in packages

**Comment by jbackman** [comment]
I can verify it is not a browserify issue. When I run the "node nexe.js", there is no issue.

**Comment by gravaton** [comment]
It seems like this issue is specific to using the cluster module in this environment combined with how Browserify packages the Javascript.  More details will be forthcoming!

**Comment by calebboyd** [comment]
Fixed in #308

**Comment by calebboyd** [comment]
Bundlers are now optional
